subprojects/docs/src/docs/release/notes.md

```diff
@@ -117,24 +103,10 @@ testing {
 }
 ```
 
-#### Dependency verification metadata improvements
+For more information about warning modes, see [JVM test suite](userguide/jvm_test_suite_plugin.html).
 
-The following nodes with dependency verification metadata file `verification-metadata.xml` now support a `reason` attribute:
 
-- the `trust` xml node under `trusted-artifacts`
-- the `md5`, `sha1`, `sha256` and `sha512` nodes under `component`
-
-#### Dependency verification CLI improvements
-
-You can now use the `export-keys` flag to export all already trusted keys:
-
-```asciidoc
-./gradlew --export-keys
-```
-
-For more information, see [Exporting keys](userguide/dependency_verification.html#sec:local-keyring).
-
-### Kotlin DSL improvements
+###  Kotlin DSL
```

Comment:
@eskatos I think this section should be after the JVM? Willing to move it elsewhere, but that seems logical to me.

Reply:
Works for me :+1: